Formula/gloo-ctl.rb

```diff
@@ -0,0 +1,31 @@
+class GlooCtl < Formula
+  desc "Envoy-Powered API Gateway"
+  homepage "https://gloo.solo.io"
+  url "https://github.com/solo-io/gloo.git",
+      :tag      => "v0.8.3",
+      :revision => "0e749d0fd9b41f8606dca85b70f0a9981cb0d5ca"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/solo-io/gloo"
+    dir.install buildpath.children - [buildpath/".brew_home"]
+
+    cd dir do
+      # Make binary
+      system "dep", "ensure", "-vendor-only"
+      system "make", "glooctl", "TAGGED_VERSION=v0.8.3", "RELEASE=false"
```

Comment:
```suggestion
      system "make", "glooctl", "TAGGED_VERSION=v#{version}", "RELEASE=false"
```

Reply:
Why `RELEASE=false`?

Reply:
RELEASE flag is tied to our internal release system. we need to think about refactoring makefile.